mz_zip_rw.c

```diff
@@ -720,8 +720,10 @@ int32_t mz_zip_reader_entry_save_file(void *handle, const char *path) {
 
     /* If symbolic link then properly construct destination path and link path */
     if (mz_zip_entry_is_symlink(reader->zip_handle) == MZ_OK) {
-        mz_path_remove_slash(pathwfs);
-        mz_path_remove_filename(directory);
+        if (pathwfs[strlen(pathwfs) - 1] == MZ_PATH_SLASH_UNIX) {
```

Comment:
Perhaps we can use `mz_path_has_slash` here?

Reply:
done, I also folded it into the previous `if`